### PR TITLE
Restricted login stuff

### PR DESCRIPTION
**Pull request by ethanphunter**
Resolves: https://github.com/ethanphunter/PointOfSaleProject/issues/67
Also Resolves: https://github.com/ethanphunter/PointOfSaleProject/issues/81

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -12,55 +12,45 @@
 products = []
 
 app.config.update(dict(
-    #DEBUG=True,
-    SECRET_KEY= os.environ["Key"]))
+    DEBUG=True,
+    SECRET_KEY= "Key"))
```
Comment (calvinbrown085) [inline main.py]: @ethanphunter Key problem

**Comment by calvinbrown085** [comment]
@ethanphunter  looks good other than my comment 👍

**Comment by ethanphunter** [comment]
@calvinbrown085 Ok that should save us much grief when running locally

**Comment by calvinbrown085** [comment]
👍

**Pull request merged**